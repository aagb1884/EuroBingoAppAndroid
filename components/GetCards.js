import { StyleSheet, View, Pressable, Text } from 'react-native';


export default function GetCards({ onPress }) {

  return (
        <View style={[styles.buttonContainer, { borderWidth: 4, borderColor: "#ffd33d", borderRadius: 18 }]}>
            <Pressable
              style={[styles.button, { backgroundColor: "yellow" }]}
              onPress={onPress}
            >
              <Text style={[styles.buttonLabel, { color: "#25292e" }]}>New Cards</Text>
            </Pressable>
          </View>
      );
    }

      const styles = StyleSheet.create({
        buttonContainer: {
          width: 340,
          height: 68,
          marginTop: 20,
          marginHorizontal: 20,
          alignItems: 'center',
          justifyContent: 'center',
          padding: 3,
        },
        button: {
          borderRadius: 10,
          width: '100%',
          height: '100%',
          alignItems: 'center',
          justifyContent: 'center',
          flexDirection: 'row',
        },
         buttonLabel: {
          color: '#fff',
          fontSize: 18,
          fontWeight: "bold",
          textAlign: 'center',
        },
      });
      