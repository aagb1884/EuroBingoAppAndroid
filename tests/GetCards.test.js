import React from 'react';
import renderer from 'react-test-renderer';

import GetCards from '../components/GetCards';

describe('<GetCards />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<GetCards />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});
