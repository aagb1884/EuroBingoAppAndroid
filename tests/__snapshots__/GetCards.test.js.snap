// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetCards /> renders correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "height": 68,
        "justifyContent": "center",
        "marginHorizontal": 20,
        "marginTop": 20,
        "padding": 3,
        "width": 340,
      },
      {
        "borderColor": "#ffd33d",
        "borderRadius": 18,
        "borderWidth": 4,
      },
    ]
  }
>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      [
        {
          "alignItems": "center",
          "borderRadius": 10,
          "flexDirection": "row",
          "height": "100%",
          "justifyContent": "center",
          "width": "100%",
        },
        {
          "backgroundColor": "yellow",
        },
      ]
    }
  >
    <Text
      style={
        [
          {
            "color": "#fff",
            "fontSize": 18,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          {
            "color": "#25292e",
          },
        ]
      }
    >
      Get Cards
    </Text>
  </View>
</View>
`;
